client/src/Pages/v1/StatusPage/Create/Components/Tabs/Settings.jsx

```
@@ -168,6 +169,118 @@ const TabSettings = ({
+						</Typography>
+						<Typography component="p">
+							{t("statusPageCreateCustomJavaScriptDescription")}
+						</Typography>
+					</Stack>
+					<Stack gap={theme.spacing(6)}>
+						<TextField
+							name="customJavaScript" // <--- CRITICAL
+							label={t("customJavaScript")}
+							color="warning" 
+    						helperText="⚠️ DANGER: Scripts added here will execute for all visitors. Only add code from trusted sources (e.g., Google Analytics)."
+							multiline
```

Biome (2.1.2)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)